web/src/beta/lib/core/Map/Layers/utils.ts

```diff
@@ -5,11 +5,10 @@ export const deepAssign = <O extends Record<string, any>>(obj: O, src: O) => {
     Object.entries({ ...src, ...obj })
       .map(([k, v]): [string, any] | undefined => {
         const srcV: unknown = src[k];
-        if (Object.hasOwn(src, k) && (srcV === undefined || srcV === null)) {
+        if (Object.hasOwn(src, k) && srcV === undefined) {
           return undefined;
         }
-
-        if (srcV === undefined) {
+        if (srcV === undefined || (isObject(srcV) && Object.keys(srcV).length === 0)) {
```

Comment:
When override with emtpy object like `marker: {}` it will compute correctly for `overriddenLayer`, but compute incorrectly for `computedLayer` (the `marker` also become {}).

Reply:
I revert this part since the previous change already solved the problem. And this is changing the behavior so better keep it as is.